taskcluster/ci/push-addon/kind.yml

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - mozillavpn_taskgraph.transforms.requires_level:transforms
+    - mozillavpn_taskgraph.transforms.release_index:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - addons
+
+jobs:
+  stage/deploy:
+      description: "Addons Stage Deploy"
+      treeherder:
+          symbol: D
+          kind: build
+          tier: 1
+          platform: addons/all
+      worker-type: b-linux
+      requires-level: 3
+      add-index-routes: addons # Name of the index
+      scopes:
+         - 'secrets:get:project/mozillavpn/tokens'
+      dependencies:
+        build: "addons-bundle"
+      fetches:
+        build:
+            - artifact: addons.zip
+      worker:
```

Comment:
Before landing please add `chain-of-trust: true` to `worker` ([ex](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/taskcluster/ci/repackage/kind.yml#L32)). The beetmover worker needs this task to generate a `chain-of-trust.json` artifact